discovery/src/main/scala/akka/discovery/SimpleServiceDiscovery.scala

```diff
@@ -3,6 +3,7 @@
  */
 package akka.discovery
 
+import java.lang.invoke.MethodHandles
```

Comment:
```suggestion
```

Reply:
yep, just notice it. It gets imported when I try to code completion Lookup